apps/charterafrica/src/lib/data/rest/index.js

```diff
@@ -53,6 +53,12 @@ export const api = {
   findPage,
 };
 
+export async function getOrganizationId() {
+  const data = await findGlobal("datasets");
+  const { organizationId } = data;
+  return organizationId;
+}
+
```

Comment:
Not sure this is necessary... both pages via lib/data and APIs have access to `payload` object and can hence fetch the organisation `id`/`slug` _**before**_ passing it to `openAfrica` lib.

The `openAfrica` lib shouldn't need to know anything about data/rest, etc.